concept/BUILD

```diff
@@ -41,13 +41,17 @@ rust_library(
         "@crates//:regex",
         "@crates//:serde",
     ],
+    proc_macro_deps = [
+        "@crates//:paste",
```

Comment:
I use it to generate multiple functions in a single macro for "smart" constraints checking. See `operation_time_validation` for `type_`.